generative/networks/nets/vqvae.py

```diff
@@ -442,8 +442,10 @@ def forward(self, images: torch.Tensor) -> tuple[torch.Tensor, torch.Tensor]:
 
         return reconstruction, quantization_losses
 
-    def encode_stage_2_inputs(self, x: torch.Tensor) -> torch.Tensor:
+    def encode_stage_2_inputs(self, x: torch.Tensor, non_quantized: bool = False) -> torch.Tensor:
```

Comment:
can we flip the argument name to `quantized` and default it to `True`? I think it's easier to understand then the double-negative `non_quantized` defaulting to `False`